.github/workflows/spell-check-comments.yml

```diff
@@ -0,0 +1,49 @@
+name: Spell Checking Code Comments
+
+on: [push,pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  spell:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.9
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install git+https://github.com/SimpleITK/CommentSpellCheck@main
```

Comment:
We should not use an untagged version. Also installing over git is slow. Use the a package from pypi.

Reply:
OK, I haven't yet gotten it into pypi.  That is the next thing to do.